docs/gherkins/DeletingSessions.feature

```diff
@@ -2,22 +2,22 @@ Feature: Sessions can be deleted when they are no longer valuable.
 
   See the following files for more information:
 
-  - [Answerers](Answerers.feature)
-  - [Questioners](Questioners.feature)
-  - [Questions](Questions.feature)
+  - [Moderators](Moderators.feature)
+  - [Proposers](Proposers.feature)
+  - [Proposals](Proposals.feature)
   - [Sessions](Sessions.feature)
 
   Scenario: Authorization
-    * The Answerer for a Session can delete that Session
-    * Other Answerers cannot delete that Session
-    * Questioners cannot delete any Session
+    * The Moderator for a Session can delete that Session
+    * Other Moderators cannot delete that Session
+    * Proposers cannot delete any Session
 
   Scenario: Deleting a Session
     Given a Session
     When that Session is deleted
-    Then all of its Questions are deleted
-    And all of its Questioners are deleted
-    And the Answerer is deleted
+    Then all of its Proposals are deleted
+    And all of its Proposers are deleted
+    And the Moderator is deleted
```

Comment:
To me it doesn't actually matter. In my haskell implementation the user and the role are synonymous, but there's no reason other implementations can't have those concepts be separate. For that reason that this is ambiguous regarding how that works is something I actually want. Do you think this is biased one way or another?